src/Array/Extra.elm

```diff
@@ -236,20 +236,17 @@ and collect the result in a `List`.
             >> Html.div []
 
 -}
-indexedMapToList :
-    (Int -> element -> mappedElement)
-    -> (Array element -> List mappedElement)
-indexedMapToList mapIndexedElement =
-    \array ->
-        array
-            |> Array.foldr
-                (\element ( i, listSoFar ) ->
-                    ( i - 1
-                    , listSoFar |> (::) (mapIndexedElement i element)
-                    )
+indexedMapToList : (Int -> a -> b) -> (Array a -> List b)
```

Comment:
Last parenthesis should be necessary, no?

Reply:
[testing github integration in vscode]
```suggestion
indexedMapToList : (Int -> a -> b) -> Array a -> List b
```